### ISSUE TITLE
/models/official/resnet/cifar10_main.py doesn't save mean and var in batch normalization

### ISSUE THREAD

**Issue by timoycao** [tensorflow/models#6313]
I convert the checkpoint trained by official/resnet/cifar10_main.py to pb model and I cannot get the mean and variance in batch normalization node. Also training option is still True. How can I get the model with mean and variance in batch_norm and set training option to False in official model?

**Comment by tensorflowbutler** [comment]
Thank you for your post. We noticed you have not filled out the following field in the issue template. Could you update them if they are relevant in your case, or leave them as N/A? Thanks.
What is the top-level directory of the model you are using
Have I written custom code
OS Platform and Distribution
TensorFlow installed from
TensorFlow version
Bazel version
CUDA/cuDNN version
GPU model and memory
Exact command to reproduce

**Comment by spxcds** [comment]
I met the same problem, did you solved that? @timoycao

**Comment by timoycao** [comment]
> I met the same problem, did you solved that? @timoycao

I didn't solve this problem in official script. But I find an alternative solution.
Create two models: one is used for training and another is for testing. Two models share the same variables. The only difference between two models is option 'is_training'. Saving the testing model will solve the problem.

**Comment by ymodak** [comment]
@timoycao Can we close this issue now that you have found a workaround?
